lib/PuppeteerSharp/DeviceDescriptors.cs

```diff
@@ -0,0 +1,741 @@
+﻿using System.Collections.Generic;
```

Comment:
this is basically the file [`DeviceDescriptors`](https://github.com/GoogleChrome/puppeteer/blob/v1.0.0/DeviceDescriptors.js)